composer.json

```diff
@@ -26,7 +26,8 @@
     },
     "require-dev": {
         "symfony/yaml": "~2.3|~3.0",
-        "phpunit/phpunit": "^6.0"
+        "phpunit/phpunit": "^6.0",
+        "phpstan/phpstan": "^0.7"
```

Comment:
@ondrejmirtes the installation on both `doctrine/annotations` and `doctrine/common` is different from what you're proposing here, is there any reason for that?